### DISCUSSION TITLE
Can't get value from AggregationResult

### DISCUSSION THREAD

**Discussion by ulpcan** [xetorthio/jedis#3244] [Q&A]
### Expected behavior

Aggregation Result casting List when using REDUCE TOLIST

### Actual behavior

I will be able to parse LIST using AggregationResult

### Steps to reproduce:

1. There is an error when getting AggregationResult using: FT.AGGREGATION query with REDUCE TO LIST

https://ibb.co/X44sPDT
https://ibb.co/kGgM9zq

Error:

https://ibb.co/HnVPWr8

There are 3 options avaliable. getString, getLong and getDouble for Row.class operations so in this case I have used getString but it should be getList or smt.

Full query:

```
FT.AGGREGATE trackerkafkastreams.model.redis.RedisDiscoverTrackDocumentIdx "@age:[99 200]" LOAD * GROUPBY 1 @trackId REDUCE TOLIST 1 @trackId AS trackIds REDUCE TOLIST 1 @userId AS userIds REDUCE TOLIST 1 @city AS cities REDUCE TOLIST 1 @trackName AS trackName REDUCE TOLIST 1 @artistName AS artistName REDUCE MAX 1 @lastUpdated AS lastUpdatedAt LIMIT 0 10 SORTBY 2 @lastUpdatedAt DESC

```
#### Jedis version:
4.3.1
#### Redis version:
6.2.6
#### Java version:
17

**Comment by sazzad16** [comment]
Issue #3245 created.
